packages/components/Menu/src/MenuPopover/useMenuPopover.ts

```diff
@@ -47,6 +50,42 @@ export const useMenuPopover = (_props: MenuPopoverProps): MenuPopoverState => {
     setPopoverHoverOutTimer(timer);
   }, [openOnHover, setOpen, setPopoverHoverOutTimer]);
 
+  const onKeyDown = React.useCallback(
+    (e) => {
+      onKeyDownProp && onKeyDownProp(e);
+
+      // Mark key events that move selection as handled.
+      // These key events are handled on the native side.
+      switch (e.nativeEvent.key) {
+        case 'ArrowUp':
+        case 'ArrowDown':
+        case 'Tab':
+        case 'Home':
+        case 'End':
+          e.stopPropagation();
+      }
```

Comment:
Might be worth putting these in a `Platform === 'win32'` check, if just for clarity. The events only bubble on the JS side on win32. On macOS these events never get sent to JS, as they are handled natively by FocusZone. 

I'm a little confused about the win32 side though. iirc, every keyboard event is sent to JS, and MenuV1 doesn't do any custom handling for any of these events. Instead they are handled by native netui, which moves focus, which moves selection. 

Is the intention that the client can override these events on Popover to do something on the JS side, but also still have native do it's thing?

Reply:
Without calling `e.stopPropogation()`, does the event just keep walking up the JS tree even if native already handled it? That seems bad..

Reply:
I take back comment on platform check. If anything this is extra safety so yay

Reply:
> Without calling `e.stopPropogation()`, does the event just keep walking up the JS tree even if native already handled it? That seems bad..

Yeah, this is basically what's happening.